### DISCUSSION TITLE
RT, BDM or MINI elements

### DISCUSSION THREAD

**Discussion by SteinStoter** [evalf/nutils#779] [Q&A] [answered]
I'd like to make use of either one of these three elements on an unstructured triangular mesh. Is something like that implemented or easy to do?

If no native support exists, then maybe the MINI is the most straightforward to do manually: it is a simple P1 velocity element with a cubic bubble function. How would I go about implementing that, and is that feasible given a limited timeframe?

**Comment by gertjanvanzwieten** [comment, marked as answer]
The MINI element is readily implemented as "bubble" (cubic in 2D, quartic in 3D), the other types are not currently implemented. If you can make do with the former then feel free to create issues for RT, BDM and any other type that might be of interest so that we can place them on the roadmap for future implementation.
